packages/calypso-e2e/src/lib/pages/plans-page.ts

```diff
@@ -17,31 +22,41 @@ const selectors = {
 	activeNavigationTab: ( tabName: PlansPageTab ) =>
 		`.is-selected.section-nav-tab:has-text("${ tabName }")`,
 
-	actionButton: ( { plan, buttonText }: { plan: Plan; buttonText: PlanActionButton } ) => {
+	actionButton: ( { plan, buttonText }: { plan: LegacyPlans; buttonText: PlanActionButton } ) => {
 		const viewportSuffix = envVariables.VIEWPORT_NAME === 'mobile' ? 'mobile' : 'table';
 		return `.plan-features__${ viewportSuffix } >> .plan-features__actions-button.is-${ plan.toLowerCase() }-plan:has-text("${ buttonText }")`;
 	},
 
-	// Plans view
-	plansGrid: '.plans-features-main',
+	// Legacy plans view
+	legacyPlansGrid: '.plans-features-main',
+
+	// Overhauled plans view
+	overhauledPlansGrid: 'div.plans > table',
+	upgradeToProButton: 'th button.is-primary',
```

Comment:
Can we not use button text due to potential i18n usage? Otherwise, I might opt for keying off the button text here, but that's a 100% optional suggestion! 👍

Reply:
Right, I chose not to key off the button text for i18n concerns.

For English, there are at least two variants:
- `Start with Pro` (en-US)
- `Try Pro risk-free` (en-GB)

The user account (`simpleSiteFreePlanUser`) uses the en-GB language and I figured it would be too jarring for the test writer (whom is likely using en-US) to see an unexpectedly different string, so I chose to go with the CSS selector.